### PR TITLE
bump versions

### PR DESCRIPTION
**Pull request by Surekh**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3398,12 +3398,14 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#a146d20d1ce4ad8ecdbae70579be6d8a7889375b",
+      "version": "github:BrightspaceHypermediaComponents/organizations#2d07b109e989ad8a2e727d436b1782a530a3f918",
```
Comment (Surekh) [inline package-lock.json]: @m6jones - FYI

File: package-lock.json [hunk 2]
```diff
@@ -3494,7 +3496,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#a63004de4a9bf1da10bd858e8f998b10109cc0f6",
+      "version": "github:Brightspace/d2l-rubric#ab05b2c7c56040b3b15b1e298c48f6e9c5aa54fd",
```
Comment (Surekh) [inline package-lock.json]: @kevinwang-d2l - FYI

File: package-lock.json [hunk 3]
```diff
@@ -3602,7 +3605,7 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:BrightspaceHypermediaComponents/sequences#e9d835bcecdf9e5cdf8829656f944f709b627c40",
+      "version": "github:BrightspaceHypermediaComponents/sequences#db30dc07f431f645d58844a6457e022cf9fe1c25",
```
Comment (Surekh) [inline package-lock.json]: @ChrisKraljevicD2L - FYI

File: package-lock.json [hunk 4]
```diff
@@ -23989,7 +23992,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#4a28b726960fed9c99f53267fb119556f45fb370",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#2267b744a1d77e6525581403f53a2f1a19713dfe",
```
Comment (Surekh) [inline package-lock.json]: @AllanKerr - FYI

Comment (AllanKerr) [reply package-lock.json]: Thanks!

**Pull request merged**